frontend/src/components/ui/multi-select.tsx

```
@@ -0,0 +1,301 @@
+                  </div>
+
+                  <Separator />
+
+                  {/* Individual options */}
+                  <div className="p-1 max-h-60 overflow-auto">
+                    {filteredSuggestions.map((suggestion) => {
+                      const isSelected = value.includes(suggestion);
+                      return (
+                        <div
+                          key={suggestion}
+                          role="option"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<option>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)